client/cli/src/commands/runcmd.rs

```diff
@@ -93,13 +94,19 @@ pub struct RunCmd {
 	#[structopt(long = "unsafe-rpc-external")]
 	pub unsafe_rpc_external: bool,
 
-	/// Don't deny potentially unsafe RPCs when listening on external interfaces.
+	/// RPC methods to expose.
 	///
-	/// Default is false. This allows exposing RPC methods publicly (same as `--unsafe-{rpc,ws}-external` )
-	/// but will allow doing so even on validator nodes, which is prohibited by default.
-	/// Please do this if you know what you're doing.
-	#[structopt(long = "unsafe-rpc-expose")]
-	pub unsafe_rpc_expose: bool,
+	/// Defaults to `Safe` (allow only a safe subset of RPC methods) if RPC is
+	/// served externally, e.g. when `--{rpc,ws}-external` is passed,
+	/// or `Unsafe` (allows every RPC method to be called) otherwise.
+	#[structopt(
+		long = "rpc-methods",
+		value_name = "METHOD SET",
+		possible_values = &RpcMethods::variants(),
+		case_insensitive = true,
+		default_value = "Default"
+	)]
```

Comment:
```suggestion
	#[structopt(
		long,
		value_name = "METHOD SET",
		possible_values = &RpcMethods::variants(),
		case_insensitive = true,
		default_value = "Default"
	)]
```
You don't need this, it is automagically done for you

Reply:
I'd keep the `long = ...` to be consistent with everywhere else here, unless we want to slowly migrate away from this explicit annotation and do this for every option a single commit

Reply:
It's already not consistent (see RunCmd) so I think you can safely use the short syntax or make a commit to make them all consistent if you want (I personally don't mind inconsistency for such a small thing).